app/livechat/server/api/v1/config.js

```diff
@@ -10,16 +10,18 @@ API.v1.addRoute('livechat/config', {
 			check(this.queryParams, {
 				token: Match.Maybe(String),
 				department: Match.Maybe(String),
+				businessUnitIds: Match.Maybe(String),
 			});
 			const enabled = Livechat.enabled();
 
 			if (!enabled) {
 				return API.v1.success({ config: { enabled: false } });
 			}
 
-			const config = Promise.await(settings());
+			const { token, department, businessUnitIds } = this.queryParams;
+
+			const config = Promise.await(settings(businessUnitIds));
```

Comment:
Have you thought about passing an object instead of a single parameter?

Reply:
Since it was just a single param, so I initially went like this... but let me change it